### ISSUE TITLE
We might want to also test management commands 

### ISSUE THREAD

**Issue by colons** [colons/instant-coverage#17]
Big updates recently run on [nkd.su](https://github.com/colons/nkdsu) have brought failure modes outside of instant coverage's scope to my attention, many of them to do with management commands breaking.

Instant coverage's testing ethos could have caught this. Mocking management commands is potentially more complicated than mocking requests, but it might be prudent to offer to cover them. I'm not sure what this would look like, but if Django offers a way to list management commands, it could fit our model pretty well.

**Comment by homsar** [comment]
> but if Django offers a way to list management commands, it could fit our model pretty well

Something like `django.core.management.get_commands()`?

**Comment by colons** [comment]
yeah, that looks right, and it'd be easier to handle exclusions than we currently do for url includes

i guess the other thing to sort is how to make this work here? i think it's obvious it'd have to be a new test class rather than another mixin, but i feel like that makes it distinct enough from instant coverage's current features that it should maybe be a different package entirely
